src/main.ts

```diff
@@ -110,7 +117,9 @@ const createWindow = (filePath?: string): BrowserWindow => {
   // Open the DevTools.
   // mainWindow.webContents.openDevTools()
 
-  newWindow.show()
+  if (!reuse) {
+    if (!process.env.HEADLESS) newWindow.show()
```

Comment:
Weren't you telling me that headless testing was breaking things for electron-playwright?

Reply:
I remember saying that but in retrospect I think I could be wrong. In any case this is useful for Mac devs that want to test without window jank by simply HEADLESS=true or whatever :)